python/dgl/data/fraud.py

```diff
@@ -0,0 +1,152 @@
+import numpy as np
+
+from .utils import save_graphs, save_info, load_graphs, load_info, _get_dgl_url
+from ..convert import from_scipy
+from .dgl_dataset import DGLBuiltinDataset
+
+class FraudDataset(DGLBuiltinDataset):
+    """Fraud node prediction dataset.
+
+    The dataset includes two homogeneous multi-relational graphs extracted from 
+    Yelp and Amazon where nodes represent fraudulent reviews or fraudulent reviewers.
+    It was first proposed in a CIKM'20 paper <https://arxiv.org/pdf/2008.08692.pdf> and 
+    has been used by a recent WWW'21 paper <https://ponderly.github.io/pub/PCGNN_WWW2021.pdf> 
+    as a benchmark. Another paper also takes the dataset as an example to study the 
+    non-homophilous graphs. This dataset is built upon industrial data and has rich 
+    relational information and unique properties like class-imbalance and feature 
+    inconsistency, which makes the dataset be a good instance to investigate how GNNs 
+    perform on real-world noisy graphs.
+
+    Parameters
+    ----------
+    url : str
+        URL to download the raw dataset
+    raw_dir : str
+        Specifying the directory that will store the
+        downloaded data or the directory that
+        already stores the input data.
+        Default: ~/.dgl/
+    save_dir : str
+        Directory to save the processed dataset.
+        Default: the value of `raw_dir`
+    """
+    file_urls = {
+        'yelp' : 'dataset/FraudYelp.zip',
+        'amazon' : 'dataset/FraudAmazon.zip'
+    }
+    relations = {
+        'yelp' : ['net_rsr', 'net_rtr', 'net_rur'],
+        'amazon' : ['homo', 'net_upu', 'net_usu', 'net_uvu']
+    }
+    file_names = {
+        'yelp' : 'YelpChi.mat',
+        'amazon' : 'Amazon.mat'
+    }
+    
+    def __init__(self, name, raw_dir=None, random_seed=2, train_size=0.7, val_size=0.1):
+        url = _get_dgl_url(self.file_urls[name])
+        self.seed = random_seed
+        self.train_size = train_size
+        self.val_size = val_size
+        super(FraudDataset, self).__init__(name=name,
+                                           url=url,
+                                           raw_dir=raw_dir)
+
+    def process(self):
+        # process raw data to graphs, labels, splitting masks
+        file_path = os.path.join(self.raw_path, self.file_names[self.name])
+        
+        data = io.loadmat(file_path)
+        node_features = torch.from_numpy(data['features'].todense())
+        node_labels = torch.from_numpy(data['label'])
+        
+        N = node_labels.shape[0]
+        
+        graphs = []
+        for relation in self.relations[self.name]:
+            g = from_scipy(data[relation])
+            graphs.append(g)
+        self.graphs = graphs
+        
+        self.feature = node_features
+        self.label = node_labels
+        self._random_split(self.feature, self.label, self.seed, self.train_size, self.val_size)
+
+    def __getitem__(self, idx):
```

Comment:
You need to explain what is stored in self.graphs. There are multiple graphs, each for one relation type.
Furthermore, it is better to give an example of how to use these graphs.

Reply:
Comments on __getitem__ and examples have been added.